vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -1,14 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
-    <ProjectConfiguration Include="Debug|x86">
```

Comment:
Is that being replaced with another config?

Reply:
The actual config is `Win32`. This was a hack to help the output folders path use `x86` instead.
This caused problems when running tests or manipulating nuget packages within the IDE.

This has been fixed, and this hack is no longer needed.